index.d.ts

```diff
@@ -792,7 +794,7 @@ declare module 'egg' {
      *
      * @member {Object} Context#locals
      */
-    locals: any;
+    locals: IContextLocals;
```

Comment:
It maybe need change to `IApplicationLocals  & IContextLocals` ? Application locals was assigned to context locals, see https://github.com/eggjs/egg/blob/master/app/extend/context.js#L157

Reply:
Yes you are right.
I will commit a new update later today.